### ISSUE TITLE
ChinaVine (chinavine)

### ISSUE THREAD

**Issue by lsat12357** [OregonDigital/OD2-migration#534]
**Item Count:** 19319 items

**Item Types:** Image/Document/Video/Generic

**Access Restrictions:**

**Complex Objects:**231

**pid List:** https://github.com/OregonDigital/OD2-migration/blob/master/chniavine/
list generated 20220827

**Comment by lsat12357** [comment]
ugh. It looks like there are 93 assets that use http://opaquenamespace.org/ns/repository/ShandongUniversityofArtandDesign for institution which is not legit as institution on OD2. there is however, a term for that place in opaquenamespace/ns/creator. Swap them?

**Comment by sseymore** [comment]
Please use http://id.loc.gov/authorities/names/n81054085

**Comment by lsat12357** [comment]
there are a handful of uris that are not getting fetched. For most of these, I think the terms need to be created still. This list generated from cpd, however similar list is on the batch ao ticket
http://opaquenamespace.org/ns/subject/MudDolls
http://opaquenamespace.org/ns/subject/Mudhttp://vocab.getty.edu/aat/300211037
http://vocab.getty.edu/aat/300053829s
http://opaquenamespace.org/ns/subject/DoughFigures
http://opaquenamespace.org/ns/subject/BristleDolls
http://opaquenamespace.org/ns/subject/PuHuiPainting
http://opaquenamespace.org/ns/subject/ClothTigers
http://opaquenamespace.org/ns/subject/FoldedPaperFigures
other than label cleanup which I suggest can happen post-migration, I think the chinavine batches are ready for QA.

**Comment by sseymore** [comment]
I'll look into these.

**Comment by lsat12357** [comment]
I just noticed that we somehow migrated 19321 instead of 19319. hmm.

**Comment by sseymore** [comment]
[chinavine_pids.txt](https://github.com/OregonDigital/OD2-migration/files/9609022/chinavine_pids.txt)

@lsat12357 pids to review for QA attached.

**Comment by sarahfish07** [comment]
**QA** 

Overall:

- [ ] All works are migrated- original number of works and OD2 works match (Migration)
- [ ] Check that a minimum of 5 works are accessible in browse everything/all
- [ ] Check that a minimum of 5 works are accessible in by clicking on them in  browse everything/all
- [ ] If collections have different Hyrax types (Documents, Images, Video, Audio, etc.), check 5 of each type, if possible. 
- [ ] Confirm works are in the correct Admin set.

Search/indexing/faceting check:

- [ ] Works and collections appear in browse everything/all when searched with corresponding metadata
- [ ] Thumbnails are appearing for works; the collection has the correct icon.
- [ ] Keyword searching appears in browse everything/all and keywords in title, description, full text appear with hit highlighting for works and collections

Facets:
- [ ] Controlled and uncontrolled facets appear and are populated by data from works (When checking facets, check the core facets in the search results and the collection-specific facets on the collection homepage.)
- [ ] Labels are fetching correctly
- [ ] URIs are resolving correctly

Work pages: 
- [ ] The digital object(s) appear in viewer
- Files are attached
- Derivatives were created (both automated check from Migration and visual check from Metadeities)
- [ ] Metadata is also there in work and viewer
- Collection information is also on work page
- Confirm metadata is in IIIF viewer
- [ ] Linked Data:
- Labels are fetching correctly
- URIs are resolving correctly
- [ ] Download
- If high res download, double-check
- [ ] Copyright 
- Confirm copyright logo matches statement

Collection pages: 
- [ ] Collection pages are there
- [ ] Collection description/blurb is there
- [ ] Facets are there

- [ ] Restrictions: Double-check restricted collections and works (VPN)

- [ ] Bulk Approve (step 11)

- [ ] Collection-level QA: Compare number of assets on OD2 with total in original pidlist (step 12)

**Comment by sarahfish07** [comment]
@lsat12357 
Broken links:
https://prod.oregondigital.org/concern/images/df67v049n
https://prod.oregondigital.org/concern/images/df70tv409
https://prod.oregondigital.org/concern/images/df67vg445
https://prod.oregondigital.org/concern/images/df70tv256
https://prod.oregondigital.org/concern/images/df67tv88b
https://prod.oregondigital.org/concern/images/df70v838x
https://prod.oregondigital.org/concern/images/df67v2709
https://prod.oregondigital.org/concern/images/df70zh03s
https://prod.oregondigital.org/concern/images/df67vp89r
https://prod.oregondigital.org/concern/images/df70nr34p
https://prod.oregondigital.org/concern/images/df67v802d
https://prod.oregondigital.org/concern/images/df70v423g
https://prod.oregondigital.org/concern/images/df70wh335
https://prod.oregondigital.org/concern/images/df70wb40j
https://prod.oregondigital.org/concern/images/df70tt45g
https://prod.oregondigital.org/concern/images/df70vd95c
https://prod.oregondigital.org/concern/images/df70nr83m
https://prod.oregondigital.org/concern/images/df71hm88m
https://prod.oregondigital.org/concern/images/df67tp18m
https://prod.oregondigital.org/concern/images/df70np73z
https://prod.oregondigital.org/concern/images/df70xb959

**Comment by sarahfish07** [comment]
NOTE: I double checked existence of errors listed in QA results above.

Checking links:
https://prod.oregondigital.org/concern/images/df67v049n
https://prod.oregondigital.org/concern/images/df70tv409
https://prod.oregondigital.org/concern/images/df67vg445
https://prod.oregondigital.org/concern/images/df70tv256
https://prod.oregondigital.org/concern/images/df67tv88b
https://prod.oregondigital.org/concern/images/df70v838x
https://prod.oregondigital.org/concern/images/df67v2709
https://prod.oregondigital.org/concern/images/df70zh03s
https://prod.oregondigital.org/concern/images/df67vp89r
https://prod.oregondigital.org/concern/images/df70nr34p
https://prod.oregondigital.org/concern/images/df67v802d
https://prod.oregondigital.org/concern/images/df70v423g
https://prod.oregondigital.org/concern/images/df70wh335
https://prod.oregondigital.org/concern/images/df70wb40j
https://prod.oregondigital.org/concern/images/df70tt45g
https://prod.oregondigital.org/concern/images/df70vd95c
https://prod.oregondigital.org/concern/images/df70nr83m
https://prod.oregondigital.org/concern/images/df71hm88m
https://prod.oregondigital.org/concern/images/df67tp18m
https://prod.oregondigital.org/concern/images/df70np73z
https://prod.oregondigital.org/concern/images/df70xb959

**Comment by sseymore** [comment]
@sarahfish07 I am seeing Tibetan painting styles. Can you clarify?

https://prod.oregondigital.org/catalog?f%5Bnon_user_collections_ssim%5D%5B%5D=chinavine&locale=en&q%5B%5D=Tibetan+painting+styles&search_field=all_fields

<beads by form> is actually the correct label, albeit not ideal

URIs are not fetching labels- just a note for clarity

**Comment by sarahfish07** [comment]
@sseymore 
When I go to the search filter, first I filter for the collection. Then, looking under topic, I cannot find "Tibetan painting styles". It is listed on the work pages, which does link to the search filter for it. I will send you screenshots on Teams.

**Comment by sseymore** [comment]
Yeah, after chatting and testing, Tibetan painting styles data is on work and accessible with a keyword search, but in looking at all topics for this collections, it's not showing up. To discuss.

**Comment by sarahfish07** [comment]
Number of assets matches. Passing to next step.

**Comment by sarahfish07** [comment]
Checked copyright for dates up to 1928. No updates needed.
